lib/code_teams/utils.rb

```diff
@@ -0,0 +1,15 @@
+module CodeTeams
+  class Utils
+    def self.underscore(string)
+      string.gsub('::', '/')
+        .gsub(/([A-Z]+)([A-Z][a-z])/, '\1_\2')
+        .gsub(/([a-z\d])([A-Z])/, '\1_\2')
+        .tr('-', '_')
+        .downcase
+    end
+
+    def self.demodulize(string)
+      string.split('::').last
+    end
```

Comment:
If we're only going to be adding class-level module methods to the `Utils` module, we can use `extend self` to avoid having to constantly write `def self.method_name`:

```suggestion
  module Utils
    extend self
    
    def underscore(string)
      string.gsub('::', '/')
        .gsub(/([A-Z]+)([A-Z][a-z])/, '\1_\2')
        .gsub(/([a-z\d])([A-Z])/, '\1_\2')
        .tr('-', '_')
        .downcase
    end

    def demodulize(string)
      string.split('::').last
    end
```